custom/conf/app.example.ini

```diff
@@ -295,9 +298,14 @@ SSH_KEY_TEST_PATH =
 ; Path to ssh-keygen, default is 'ssh-keygen' which means the shell is responsible for finding out which one to call.
 SSH_KEYGEN_PATH = ssh-keygen
 ; Enable SSH Authorized Key Backup when rewriting all keys, default is true
-SSH_BACKUP_AUTHORIZED_KEYS = true
+SSH_AUTHORIZED_KEYS_BACKUP = true
+; Enable SSH Authorized Principals Backup when rewriting all keys, default is true
+SSH_AUTHORIZED_PRINCIPALS_BACKUP = true
 ; Enable exposure of SSH clone URL to anonymous visitors, default is false
 SSH_EXPOSE_ANONYMOUS = false
+; Specifies the public keys of certificate authorities that are trusted to sign user certificates for authentication.
```

Comment:
Please elborate which public key format.

Reply:
This is literally copied from the sshd_config man page, where it is also not specified. https://manpages.debian.org/experimental/openssh-server/sshd_config.5.en.html#TrustedUserCAKeys

Reply:
Is it `ssh-<format> <key>`? I'd put that as example.

Reply:
Yup, it's a normal public key